rpm_spec/subpackages/manageiq-appliance

```diff
@@ -7,7 +7,7 @@ Requires: %{name}-core-services = %{version}-%{release}
 Requires: %{name}-gemset-services = %{version}-%{release}
 
 Requires: postgresql-server
-Requires: repmgr10 >= 4.0.6
+Requires: repmgr13 >= 5.2.1
```

Comment:
This can be changed later if it makes sense.

Reply:
This is the line that will force appliances to use either pg 10 or 13.  repmgr13 is repmgr built against pg 13 and will not work with pg 10.  If we want to allow appliances to use pg 10, this line would need to be removed and somehow the correct repmgr gets installed.